build/android/sign_apk.py

```diff
@@ -3,34 +3,52 @@
 # found in the LICENSE file.
 
 import argparse
+import os
 import sys
 import subprocess
 import tempfile
 
 def main():
-  argument_parser = argparse.ArgumentParser()
-  argument_parser.add_argument('zipalign_path')
-  argument_parser.add_argument('apksigner_path')
-  argument_parser.add_argument('unsigned_apk_path')
-  argument_parser.add_argument('key_path')
-  argument_parser.add_argument('key_passwd')
-  argument_parser.add_argument('prvt_key_passwd')
-  argument_parser.add_argument('key_name')
-  args = argument_parser.parse_args()
+    argument_parser = argparse.ArgumentParser()
```

Comment:
can we not use apkbuilder.py from upstream? Or at least use `FinalizeApk` from finalize_apk.py? I feel more comfortable with this stuff when we're doing the same thing as upsteam

Reply:
The files you mentioned are only appicable to `apk` files. The only place upstream tried to sign a bundle is when creating a wrapper for running bundles, and they only sign the extracted apks in that case not the bundle itself (see `//build/config/android/rules.gni#L4915`).

Reply:
ok, they looked very similar, but maybe there were some differences I didn't notice